### PR TITLE
Language Update #1

### PR DESCRIPTION
**Pull request by im-a-dweeb**
I've gone ahead and added 2 more languages along with 1 special one ;) (Ava, please don't kill me I have a family-)

- Suisse French
- Deutsch
- Special One ;)

### CODE REVIEW

File: translations/pirate-us.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    "personal_thoughts_name_label_text": {
+        "ORIGINAL-VERSION": "Personal Thoughts",
+        "translation": "Private Musings"
+    },
+
+    "personal_thoughts_name": {
+        "ORIGINAL-VERSION": "What are the oc's thoughts towards their name?",
+        "translation": "What be the OC's musings 'bout their moniker?"
+    }
+}
```
Comment (AvatarKage) [inline translations/pirate-us.json:164]: This isn't something we're looking for!

File: index.html [hunk 2]
```diff
@@ -497,6 +497,9 @@
         <script src="scripts/menus/language_menu.js"></script>
         <ul>
           <li id="en-us">🦅 English (US)</li>
+          <li id="de-de">🦅 Deutsch (DE)</li>
+          <li id="fr-ch">🇨🇭 Suisse Français (CH)</li>
+          <li id="pirate">🏴‍☠️Pirate</li>
```
Comment (AvatarKage) [inline index.html:502]: Find emotes that represents the native language appropriately or at least the country of origin. Flags don't show on all devices! Also remove the pirate as it's not what we're looking for.

File: scripts/translate.js [hunk 3]
```diff
@@ -83,6 +86,27 @@ if (EnUsButton) {
...
+        onLoadEmojis();
+    }, 300)
+})
+
+pirateButton.addEventListener("click", () => {
+    translateTo("pirate-US")
+    setTimeout(() => {
+        onLoadEmojis();
+    }, 300)
+})
```
Comment (AvatarKage) [inline scripts/translate.js:109]: Remove pirate!

**Review by AvatarKage — CHANGES_REQUESTED** [review]
I've commented on a few things that need to be addressed before this merge can be completed!

**Comment by ScuffedItalian** [comment]
I will remove those and update stuff per requested. One issue, for Suisse, their national animal isn't an emoji and CH is switzerland's region code. I have no clue what to put for Switzerland, and Deutschland's (Germany's) national animal is also an eagel so it is a little difficult to come up with a complete different one for Deutschland other than DE.

**Comment by AvatarKage** [comment]
> I will remove those and update stuff per requested. One issue, for Suisse, their national animal isn't an emoji and CH is switzerland's region code. I have no clue what to put for Switzerland, and Deutschland's (Germany's) national animal is also an eagel so it is a little difficult to come up with a complete different one for Deutschland other than DE.

That's fine then. I can find an emoji for it don't! Just leave them blank then, thank you!

**Comment by im-a-dweeb** [comment]
Alright, all of that has been done. Anything else you'd like me to change?

**Review by AvatarKage — APPROVED** [review]
Thank you!

**Pull request merged**